SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -316,6 +347,7 @@ bool FLocalDeploymentManager::TryStartSpatialService()
 	if (ServiceStartResult.Contains(TEXT("RUNNING")))
 	{
 		UE_LOG(LogSpatialDeploymentManager, Log, TEXT("Spatial service started!"));
+		ExposedRuntimeIP = RuntimeIPToExpose;
 		bSpatialServiceRunning = true;
 		return true;
 	}
```

Comment:
Should `ExposeRuntimeIP` be cleared in the instance that the service fails to start?

Reply:
I was thinking in the instance where it was changed, and then the deployment failed to start, it'll be populated with an old runtime ip, which might be confusing. I also see there's another instance where bSpatialServiceRunning is set to true, and should probably be cleared then also.